spec/.rubocop.yml

```diff
@@ -75,9 +75,17 @@ RSpec/ExpectInHook:
   Enabled: false
 
 # your naming scheme is not in possession of all the facts
-RSpec/FilePath:
+
+RSpec/IndexedLet:
+  Enabled: false
+
+RSpec/SpecFilePathFormat:
```

Comment:
I can't make a suggestion because of the deleted line, but I think the comment on line 77 belongs with the file path format cop on line 82.

Reply:
fixed with rebased changes